src/main/java/org/gridsuite/modification/server/entities/equipment/modification/DeleteAttachingLineEntity.java

```diff
@@ -0,0 +1,86 @@
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+package org.gridsuite.modification.server.entities.equipment.modification;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.gridsuite.modification.server.ModificationType;
+import org.gridsuite.modification.server.dto.DeleteAttachingLineInfos;
+import org.gridsuite.modification.server.dto.ModificationInfos;
+import org.gridsuite.modification.server.entities.ModificationEntity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+/**
+* @author bendaamerahm <ahmed.bendaamer at rte-france.com>
+*/
+@NoArgsConstructor
+@Getter
+@Entity
+@Table(name = "DeleteAttachingLine")
+public class DeleteAttachingLineEntity extends ModificationEntity {
+
+    @Column
+    private String lineToAttachTo1Id;
+
+    @Column
+    private String lineToAttachTo2Id;
+
+    @Column
+    private String attachedLineId;
+
+    @Column
+    private String replacingLine1Id;
+
+    @Column
+    private String replacingLine1Name;
+
+    public DeleteAttachingLineEntity(String lineToAttachTo1Id, String lineToAttachTo2Id, String attachedLineId, String replacingLine1Id, String replacingLine1Name) {
```

Comment:
entity should be built with a DTO.
see for instance LineAttachToVoltageLevelEntity